packages/replay/test/utils/setupReplayContainer.ts

```diff
@@ -13,8 +13,8 @@ export function setupReplayContainer({
       flushMinDelay: 100,
       flushMaxDelay: 100,
       stickySession: false,
-      sessionSampleRate: 0,
-      errorSampleRate: 1,
+      sessionSampleRate: 1,
+      errorSampleRate: 0,
```

Comment:
Is this change intentional?

Reply:
Yes, I figured it makes more sense to default this to sample as session then as error!